### ISSUE TITLE
Light-weight client

### ISSUE THREAD

**Issue by martinheidegger** [soyuka/dat-daemon#18]
Would you be okay with a PR that changes the client API to return light-weight "DAT-proxies" with the same API as DAT but where all calls go to dat-daemon over the protocol?

**Comment by soyuka** [comment]
Yes of course. Note that I've seen some other "daemon" implementations out in the wild, maybe that my approach isn't the best one! I'm open to any improvements or suggestions though!

**Comment by martinheidegger** [comment]
Hmm, not sure which you are referring to.
